### ISSUE TITLE
[CAIA A11y Research - PILOT]: Public Websites - New Federal Standardized Header and Footer P2 - Pilot Research Sessions (Jan. 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#74018]
# CAIA A11y Support
This is a child ticket of the research ticket (#73242) to track the work done for the pilot sessions.

## Assignees

- Eli M.
- Sarah K.
- Evan B.

## Description

CAIA a11ys will participate as in a pilot of the [Public Website team's New Federal Standardized Header and Footer Phase 2 research study](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73242).

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` on `desktop` and `screen magnifier` on `desktop` would behave.

### Assistive Tech Options

- **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
- **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Team Info

- Team name: Public Websites
- OCTO-DE product owner: Dave Conlon Shane Strassberg (VA Research Ops)
- Product manager: Fran Cross
- Designer: Amanda Klausmeier
- FE engineer: Randi Mays
- Researcher: Cindy Merrill
- Accessibility: Laura Flannery
- Product/team Slack channel: [sitewide-public-websites](https://dsva.slack.com/archives/C52CL1PKQ/p1703716392350969)

## Pilot Sessions
See [original comment in the research ticket 73242:](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73242#issuecomment-1899190507) SK and Eli are supporting the pilot sessions for a screen magnifier (Eli) and screen reader (SK). 
- [x] **Tuesday**, Jan. 23, 2024 @SarahKay8 `Screen Reader` from 3:30 to 5:00 p.m. ET
- [x] **Wednesday**, Jan. 25, 2024 @eli-oat `Screen Magnification` from 12-1:30 p.m. ET

_^Dates subject to change, per Cindy._

### Prototype and Conversation Guide
- [Header/Footer CodePen Prototype](https://cdpn.io/pen/debug/gOEPoyo/43fa8051200887cb97f5b154ed54c8a3) 
    - If you wanted to play around in advance, here is the link to the prototypes. 
    - Laura F. said to reach out to her if you have any questions (she made them.) 
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/header-footer/federal-standardized-header-footer/research/phase2-AT/conversation-guide.md)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] Determine date and time of sessions
- [x] Add hold to a11y Capacity Calendar
- [x] Perigean to schedule with CAIA a11ys
- [x] Add official invite to the a11y Capacity Calendar
- [x] Review the [Header/Footer CodePen Prototype](https://cdpn.io/pen/debug/gOEPoyo/43fa8051200887cb97f5b154ed54c8a3)
- [x] Review the [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/header-footer/federal-standardized-header-footer/research/phase2-AT/conversation-guide.md)
- [x] Run pilot sessions (2)
- [ ] Share findings [View a Recent Example by Jamie](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/MHV%20Cartography%20Team/Jan24-new-nav-Design-Review.md) @SarahKay8 and @eli-oat
```
